app/lib/constants/site_config.rb

```diff
@@ -283,11 +283,15 @@ module SiteConfig
         placeholder: "List of valid tags: comma separated, letters only e.g. beginners,javascript,ruby,swift,kotlin"
       },
       suggested_users: {
-        description: "Determines which users are suggested to follow to new users during onboarding (comma" \
-        "separated, letters only). Please note that these users will be shown as a fallback if no" \
+        description: "Determines which users are suggested to follow to new users during onboarding (comma " \
+        "separated, letters only). Please note that these users will be shown as a fallback if no " \
         "recently-active commenters or producers can be suggested",
         placeholder: "List of valid usernames: comma separated, letters only e.g. ben,jess,peter,maestromac,andy,liana"
       },
+      prefer_manual_suggested_users: {
+        description: "Always show the users determined above as suggested people to follow even when " \
```

Comment:
I think we should reword this description somehow such that `suggested_users` doesn't _always_ have to be rendered _above_ the `prefer_manual_suggested_users` in the config view. If it ever gets moved around in `app/views/admin/configs/show.html.erb`, then we would have to remember to change this wording, too 😅 
```suggestion
        description: "Always show suggested users as suggested people to follow even when " \
```